freq/lizard_api_connector.py

```diff
@@ -116,7 +117,7 @@ def parse(self):
                 self.results += self.json['results']
                 next_url = self.json.get('next')
                 if next_url:
-                    self.request(next_url)
+                    self.fetch(next_url)
                 else:
                     break
             except IndexError:
```

Comment:
Een "while True" met een "except indexerror" klinkt wat creatief. Regulier for loopje en een iterator gebruiken? Klinkt nu als een rare mengvorm.